File: challenges/4-fizz-buzz/python/pontes-guilherme/fizzbuzz.py
from sys import argv

def fizzbuzz(last_number):
    for number in range(1, last_number + 1):
        if number % 3 == 0 and number % 5 == 0:
            print("FizzBuzz")
        elif number % 3 == 0:
            print("Fizz")
        elif number % 5 == 0:
            print("Buzz")
        else:
            print(number)


def main(*args):
    _, *numbers = args

    for number in numbers:
        fizzbuzz(int(number))
        print("\n")


if __name__ == "__main__":
    main(*argv)

File: challenges/4-fizz-buzz/javascript/Jou/fizz-buzz.js
for (let i = 1; i <= process.argv[2]; i++) {
    if ((i % 3) === 0 && (i % 5) === 0) {
        console.log("FizzBuzz")
    } else if ((i % 5) === 0) {
        console.log("Buzz")
    } else if ((i % 3) === 0) {
        console.log("Fizz")
    } else {
        console.log(i)
    }
}
